.github/workflows/ci.yml

```
@@ -182,3 +187,75 @@ jobs:
           helm repo update
           touch kubeconfig && chmod 600 kubeconfig && echo "${{ secrets.KUBECONFIG }}" > kubeconfig
           KUBECONFIG="kubeconfig" helm --namespace="${NAMESPACE}" upgrade "${RELEASE}" jupyterhub/jupyterhub --version=0.11.1 --values config.yaml --cleanup-on-fail
+
+
+  deploy-marketplace:
+
+    needs: [build-docker-image]
+    if: github.ref == 'refs/heads/marketplace'
```

Same as above, I believe thhis should be:
```suggestion
    if: github.ref == 'refs/heads/main'
```